app/serializers/optimized_project_serializer.rb

```diff
@@ -55,7 +63,7 @@ def serialize_project(project)
           repository_license: project.repository_license,
           repository_status: project.repository_status,
           stars: project.stars,
-          versions: project.versions,
+          versions: @minimal_versions ? project.versions.map { |v| v.slice(*VERSION_ATTRIBUTES) } : project.versions,
```

Comment:
couple thoughts:
 * would anyone actually care if we dumped the extra stuff from https://github.com/librariesio/libraries.io/blob/main/app/serializers/version_serializer.rb ? 
 * it might be an even bigger win to switch to `pluck_to_hash` instead of model+serializer ?
 
 What you're dropping here it looks like is:
   * `spdx_expression` - this is not reliable, `original_license` should be used
   * `researched_at` - this looks like it was supposed to be when someone fills in spdx_expression but it doesn't look like any code sets it and we don't care about spdx_expression anyway
   * `repository_sources` - this looks like it's needed to construct release download URLs (does it need to be per-release though?). But if this is the one optional thing maybe the parameter is like `extra_data=repository_sources` to ask for it when needed?